### PR TITLE
changes for elastic v7

### PR DESCRIPTION
**Pull request by nicpottier**
Mostly just updating the library we are dependent on and tweaking a few parameters. For 8.0 we'll have to address the mapping issue.

### CODE REVIEW

File: indexer.go [hunk 1]
```diff
@@ -84,7 +84,7 @@ func CreateNewIndex(url string, alias string) (string, error) {
 	}
 
 	// initialize our index
-	createURL := fmt.Sprintf("%s/%s", url, physicalIndex)
+	createURL := fmt.Sprintf("%s/%s?include_type_name=true", url, physicalIndex)
```
Comment (nicpottier) [inline indexer.go:87]: Needed for mapping to be supported.

File: indexer.go [hunk 2]
```diff
@@ -631,10 +631,10 @@ const indexSettings = `
 const lastModifiedQuery = `{ "sort": [{ "modified_on_mu": "desc" }]}`
 
 // indexes a contact
-const indexCommand = `{ "index": { "_id": %d, "_type": "_doc", "_version": %d, "_version_type": "external", "_routing": %d} }`
+const indexCommand = `{ "index": { "_id": %d, "_type": "_doc", "version": %d, "version_type": "external", "routing": %d} }`
 
 // deletes a contact
-const deleteCommand = `{ "delete" : { "_id": %d, "_type": "_doc", "_version": %d, "_version_type": "external", "_routing": %d} }`
+const deleteCommand = `{ "delete" : { "_id": %d, "_type": "_doc", "version": %d, "version_type": "external", "routing": %d} }`
```
Comment (nicpottier) [inline indexer.go:637]: Same parameters, just no underscores anymore.

File: indexer.go [hunk 3]
```diff
@@ -660,8 +660,10 @@ type aliasCommand struct {
 // our response for finding the most recent contact
 type queryResponse struct {
 	Hits struct {
-		Total int `json:"total"`
-		Hits  []struct {
+		Total struct {
+			Value int `json:"value"`
+		} `json:"total"`
```
Comment (nicpottier) [inline indexer.go:665]: Now an object, this will have repercussions on how we query on the mailroom side.

File: go.sum [hunk 4]
```diff
@@ -1,5 +1,10 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```
Comment (rowanseymour) [inline go.sum:1]: looks like a little `go mod tidy` wouldn't go amiss

**Review by rowanseymour — APPROVED** [review]

File: go.sum (hunk 4 above)
Comment (nicpottier) [reply go.sum:1]: Thought the same, but no, these are all still there afterwards.

**Comment by nicpottier** [comment]
fixes #24

**Pull request merged**